meta-iotqa/lib/oeqa/runtime/multimedia/realsense/realsense_capture_image.py

```diff
@@ -0,0 +1,27 @@
+import time
+from oeqa.oetest import oeRuntimeTest
+
+
+class RealsenseHeadlessTest(oeRuntimeTest):
+    def test_realsense_headless(self):
+        # First make sure that uvcvideo module is loaded.
+        (status, output) = self.target.run('modprobe uvcvideo')
+        self.assertEqual(status, 0, msg="Error messages: %s" % output)
+        # Run an example test from librealsense-examples.
+        (status, output) = self.target.run('cpp-headless')
+        if status == 1:
+            # There might not be correct HW connected.
+            self.assertEqual(output, "There are 0 connected RealSense devices.", msg="Error messages: %s" % output)
+        else:
+            self.assertEqual(status, 0, msg="Error messages: %s" % output)
+        # Capture DEPTH image and checking image file present
+        (status, output) = self.target.run("/bin/cpp-headless")
+        self.assertEqual(status, 0, msg="Exe file not running: %s" % output)
+        (status, output) = self.target.run("find / -name cpp-headless-output-DEPTH.png > /tmp/log.txt")

```

Comment:
the output image is written in the "current working directory". no need to search ```/ ```